Blockchain/readme.md

```diff
@@ -52,10 +52,9 @@
 ## Solidity
 - [Remix IDE](https://remix-project.org/)
 - Solidity Tutorials :
-  - [1)Solidity With Javascript](https://youtu.be/gyMwXuJrbJQ)
-  - [2) Solidity With Python](https://youtu.be/M576WGiDBdQ) 
-  - [3) Smart Contractor Programme](https://www.youtube.com/channel/UCJWh7F3AFyQ_x01VKzr9eyA)
-  - [4) Solidity Tutorial](https://youtu.be/3g2WT2jms_k)
+- [Solidity With Javascript 31 hours Course](https://youtu.be/gyMwXuJrbJQ)
+- [Solidity With Javascript 10 hours Course](https://youtu.be/cGQHXmCS94M) 
+- [3) Solidity Tutorial](https://youtu.be/3g2WT2jms_k)
```

Comment:
The syntax of this is not matching the other resources. Please fix it.

Reply:
Just fixed the syntax.